.github/workflows/main.yml

```diff
@@ -15,14 +15,14 @@ env:
 jobs:
   python-checks:
     name: minimaltest (pre-commit)
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
       - name: Install dependencies
         run: |
-          #: Install Python 2.7 from Ubuntu 20.04 using apt-get install
+          #: Install Python 2.7 from Ubuntu 22.04 using apt-get install
```

Comment:
Should python3 be installed instead?

Reply:
The Python2 is still in the pipeline :(
So it can not be updated to Ubuntu 24.04 in which the Python2 is not supported.

Reply:
Well, shouldn't python2 be removed from the pipeline altogether, then?

Reply:
The Python2 was not removed when doing the migration from Python2 to Python3. I think there might be a reason for that.

Reply:
probably just forgot to remove it